tests/unit/lms/services/mailchimp_test.py

```diff
@@ -14,21 +14,15 @@
 
 
 class TestSendTemplate:
-    def test_it(self, mailchimp_transactional, caplog):
+    def test_it(
+        self, svc, mailchimp_transactional, mailchimp_client, caplog, sender, recipient
+    ):
         caplog.set_level(logging.INFO)
-        svc = MailchimpService(sentinel.api_key)
 
         svc.send_template(
             sentinel.template_name,
-            EmailSender(
-                sentinel.subaccount_id,
-                sentinel.from_email,
-                sentinel.from_name,
-            ),
-            EmailRecipient(
-                sentinel.to_email,
-                sentinel.to_name,
-            ),
```

Comment:
These were duplicated in lots of tests so I moved them into `sender` and `recipient` fixtures